bucket/btop-lhm.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "1.0.2",
+    "description": "btop++ for windows",
+    "homepage": "https://github.com/aristocratos/btop4win",
+    "license": "Apache-2.0",
+    "notes": [
+        "Has full GPU monitoring, etc. using Libre Hardware Monitor, DLL's included in the package.",
+        "Requires admin rights to run."
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/aristocratos/btop4win/releases/download/v1.0.2/btop4win-LHM-x64.zip",
+            "hash": "898b0725918e58926851304195e795b881abd1c2353b0537e62ea1a273777a66",
+            "extract_dir": "btop4win"
+        }
+    },
```

Nitpick: move this out to architecture in case of future 32-bit and ARM versions.

```suggestion
            "hash": "898b0725918e58926851304195e795b881abd1c2353b0537e62ea1a273777a66"
        }
    },
    "extract_dir": "btop4win",
```